bcgov/HOUS-permit-portal
# UI cleanup round3

## Description

- Adding in a nicer placeholder image for low density residential.
- Adding link to the CMS lite pages
- Remove background and make full width the box of preamble text for when submitter starts a new application

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-870


## Steps to QA

As a submitter go start a new application and you should see the new image for the low density permit type. And if you go to "My Active Permits" screen, on the permit card it should show the new image as well.

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?
